cmd/block-generator/docker/Dockerfile-indexer

```diff
@@ -11,7 +11,7 @@ RUN go build
 CMD ["/bin/sh", "-c", "\
   echo $ALGOD_NET && \
   echo $CONNECTION_STRING &&\
-  sleep 10 && \
+  sleep 5 && \
```

Comment:
This sort of thing is a common problem with docker. The framework only concerns itself with the container starting, not the service within the container. This is a solution for Postgres, but [sandbox doesn't even do it properly](https://github.com/algorand/sandbox/issues/31):
```
# Wait for postgres to start.
while ! curl http://${DB_ADDR}/ 2>&1 | grep '52'
do
  sleep 1
done
```

There are also popular bash scripts that some people like to include in their deployments, [like wait-for-it](https://github.com/vishnubob/wait-for-it).

For our purposes, I think it's better to use sleep and overshoot the startup time rather than introduce something that might break.